### PR TITLE
Refactor init

### PR DESCRIPTION
**Pull request by yenda**
Refactoring init sequence to remove event fireworks.
Incidently login is a bit faster

# note to reviewers

important: node is restarted everytime user logs in

- [x] refactor intialize account 
- [x] refactor initialize account tests
tdb.
- [x] move init, web3, notifications, signals events and functions to dedicated namespaces
- [x] refactor login flow in `accounts.login.models`

Simplified flow for login:
- event `:ui/login` is dispatched
- node is initialized with user config or default config
- `node.started` signal is received, applying `:login` fx
- `:callback/login` event is dispatched, account is changed
in datastore, web-data is cleared
- `:init/initialize-account` event is dispatched

- [x] replace event dispatches by function composition

# how to test
need extensive tests, mostly around account login, creation and recovery, change network and enable bootnodes, try transactions and web3 browsing.

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/events.cljs [hunk 1]
```diff
@@ -293,56 +309,71 @@
...
-                    :device-UUID device-UUID
-                    :semaphores semaphores)
-       console-contact
-       (assoc :contacts/contacts {constants/console-chat-id console-contact})))))
+(defn initialize-account-db [address {:keys [db web3]}]
+  (let [{:keys [accounts/accounts accounts/create contacts/contacts networks/networks
+                network network-status peers-count peers-summary view-id navigation-stack
+                status-module-initialized? status-node-started? device-UUID
+                push-notifications/initial? semaphores]
+         :or   [network (get app-db :network)]} db
```
Comment (gnl) [inline src/status_im/ui/screens/events.cljs]: This should be `:or {network ...}`

**Review by janherich — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by janherich — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]
lgtm. just remember to squash the commits

**Comment by lukaszfryc** [comment]
@yenda there is no send button in chats. Also, the command button does not work when tapped. Other than that it works fine. I did not notice any improvements in login speed.

**Comment by yenda** [comment]
@goranjovic I think I will squash it into 3, I would like to keep some granularity in case some bug slips in and we need to bisect, I built and tested at every commit so they are all good

**Comment by yenda** [comment]
@lukaszfryc I had to do a complicated rebase with the password saving feature that was merged
android: https://i.diawi.com/MysGZ2
iOS: https://i.diawi.com/ANCkok

<blockquote><div><strong><a href="https://i.diawi.com/MysGZ2">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>
<blockquote><div><strong><a href="https://i.diawi.com/ANCkok">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request merged**